modules/nw-sctp-about.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-about_{context}"]
+= Support for Stream Control Transmission Protocol (SCTP) on {product-title}
+
+If your cluster is installed on bare metal infrastructure, you can enable SCTP on the hosts in the cluster. On {op-system-first} the SCTP module is blacklisted by default.
+
+SCTP is a reliable message based protocol that runs on top of an Internet Protocol (IP) network.
```

Comment:
(We don't expand "IP" anywhere else in the docs either.)

The customers who actually need SCTP support already know exactly what it is, so it might be useful to aim this description at people who *don't* need SCTP support, to assure them that they don't need to care about it and can just skip this section. eg, maybe add something about "it is not widely used except in the telecom industry".